konf/site.yaml

```
@@ -366,6 +369,9 @@ staging:
     if ($host = 'insights.staging.ubuntu.com' ) {
       rewrite ^ https://staging.ubuntu.com/blog$request_uri? permanent;
     }
+    if ($host = 'certification.staging.ubuntu.com/' ) {
+      rewrite ^ https://ubuntu.com/certified$request_uri? permanent;
```

This should surely redirect to staging?

```suggestion
      rewrite ^ https://staging.ubuntu.com/certified$request_uri? permanent;
```